sit/launch_review_job.py

```diff
@@ -124,29 +111,39 @@ def get_container_definitions(self, role, family):
         return container.get_container_definitions()
 
     def start_tasks(self, tasks):
-        logging.info('Starting tasks')
+        logging.info('Starting tasks: {0}'.format(tasks))
         try:
             task_ids = []
             for task in tasks:
-                response = self.ecs_client.start_task(
-                    cluster=self.cluster,
-                    taskDefinition=task,
-                    containerInstances=[self.instance_arn]
-                )
+                response = self.attempt_start_task(task)
                 task_ids.append(response['tasks'][0]['taskArn'])
             return task_ids
         except Exception as e:
             self.error('Failed to run tasks', e)
 
+    def attempt_start_task(self, task, attempt=1):
+        response = self.ecs_client.run_task(
+            cluster=self.cluster,
+            taskDefinition=task
+        )
+        if attempt > 10:
```

Comment:
You mean it may succeed on the 11th attempt but we'll still fail because of the position of the check. 👍  Moving the check up.